README.md

```diff
@@ -7,11 +7,13 @@
  [![Twitter Follow](https://img.shields.io/twitter/follow/Teku_Consensys)](https://twitter.com/Teku_Consensys)
  [![GitPOAP Badge](https://public-api.gitpoap.io/v1/repo/ConsenSys/teku/badge)](https://www.gitpoap.io/gh/ConsenSys/teku)
 
-Teku is a Java implementation of the Ethereum 2.0 Beacon Chain. See the [Changelog](https://github.com/Consensys/teku/releases) for details of the latest releases and upcoming breaking changes.
+Teku is an open-source Ethereum consensus client written in Java and containing a full beacon node 
+and validator client implementation.  
```

Comment:
you can avoid this new-line.

Reply:
I saw we added a line break down in the special thanks section so I hesitated to cross the 120 chars... I removed it in the end